scripts/test_twitter_ingestion.py

```
@@ -71,7 +71,48 @@ def test_ingestion(data, data_type: str):
         
-        if response.status_code == 200:
+        # Handle async 202 response
+        if response.status_code == 202:
+            result = response.json()
+            progress_id = result.get("progress_id")
+            print(f"   ⏳ Ingestion started async (ID: {progress_id}). Polling for status...")
+            
+            # Poll for completion
+            import time
+            max_retries = 30
+            for i in range(max_retries):
```

Ruff (0.14.8)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)